Cargo.toml

```diff
@@ -20,6 +20,24 @@ members = [
     "kuksa_databroker/databroker-examples",
 ]
 
+[workspace.dependencies]
+clap = { version = "3.1.10", default-features = false, features = [
+    "std",
+    "env",
+] }
+databroker-proto = { path = "kuksa_databroker/databroker-proto" }
+prost = "0.11"
+prost-types = "0.11"
+tokio = { version = "1.17.0", features = [
+    "macros",
+    "rt-multi-thread",
+    "time",
+    "signal",
+] }
+tokio-stream = { version = "0.1.8", features = ["sync", "net"] }
+tonic = "0.8"
+tonic-build = "0.8"
+
```

Comment:
Oh, and one more thing. Since features are additive, in order for the workspace.dependencies not to inadvertently enable all the default features (see [default-features ignored](https://github.com/rust-lang/cargo/issues/11329)), `default-features = false` should also be retained in the workspace dependencies.

So at least for clap in this case (and maybe for all?)
```
clap = { version = "3.1.10", default-features = false }
```

Reply:
Shoudn't that also be the decision (responsibility) of the different packages? In the end they will need to know if they need/want the default features or not, or won't they?

Reply:
Yes exactly. So in order to give them the ability to make that decision, the workspace dependencies need to have `default-features = false`.

Otherwise, default-features will be enabled for every member regardless of what the specify themself.